app/src/main/res/values/strings.xml

```diff
@@ -79,6 +79,8 @@
     <string name="subtype_akkhor">Akkhor</string>
     <!-- Description for Ergo-L keyboard layout. -->
     <string name="subtype_ergol">Ergo-L</string>
+	<!-- Description for Hcesar keyboard layout. -->
+    <string name="subtype_hcesar">Hcesar</string>
```

Comment:
For some reason `Hcesar` layout is not shown in the layout list in Portuguese language settings.